corehq/apps/userreports/reports/README.md

```diff
@@ -0,0 +1,58 @@
+Reports Overview
+================
+
+There are two main components involved in producing a report:
+
+* the data source configuration
+* the report configuration
+
+
+The data source
+---------------
+
+The data source is usually a table in PostgreSQL, but data sources can also be ElasticSearch indexes.
```

Comment:
ES is only used by 1 data source in enikshay. I'd probably leave it out of these docs as I don't think it's every going to be used (and if it is that person should know what they're doing)

Reply:
Good to know. I had only come across Postgres data sources, but I saw the code supported ES.